src/app.py

```diff
@@ -574,7 +585,7 @@ def load_file(filename: str) -> dict:
                     [
                         "Repository Name",
                         "Dependency",
-                        "Advisory",
```

Comment:
Did you mean for this change to come in?

Reply:
Yes, it removes information from the dashboard that the CAB probably won't like. Did it in this PR so we can give the dashboard back to CAB.